components/tyk-dashboard/templates/deployment-dashboard.yaml

```diff
@@ -1,3 +1,5 @@
+{{- $isSemVer := regexMatch "^\\d+\\.\\d+\\.\\d+$" .Values.dashboard.image.tag }}
```

Comment:
This regex wont't pass for `v5.3.0-rc`.
You could use regex suggested on SemVer page itself [https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string)